templates/report_template.html

```diff
@@ -10,6 +10,13 @@
 				<p style="text-align: center; font-size: smaller;">Filtered job builds by parameter name {{header.fpn}} with value {{header.fpv}}</p>
 			{% endif %}
 		</div>
+		{% if preamble is not none %}
+			<div>
```

Comment:
Ignore the indentations and spaces
```suggestion
			{% if preamble is not none %}
                        <pre>{{ preamble }}</pre>
                     {% endif %}
```